src/applications/personalization/profile-2/components/MilitaryInformation.jsx

```diff
@@ -16,138 +16,130 @@ import facilityLocator from 'applications/facility-locator/manifest.json';
-              8:00 p.m. ET. If you have hearing loss, call TTY: 1-866-363-2883.
-            </p>
-          </AdditionalInfo>
-        </div>
-      </>
+      <AlertBox
+        isVisible
+        status="warning"
+        headline="We can’t access your military information"
+        content={
+          <p>
+            We’re sorry. We can’t access your military service records. If you
+            think you should be able to view your service information here,
+            please file a request to change or correct your DD214 or other
+            military records.
+          </p>
+        }
+      />
     );
   }
-}
 
-class MilitaryInformation extends Component {
-  render() {
-    return (
-      <>
-        <h2 tabIndex="-1" className="vads-u-margin-y--4">
-          Military information
-        </h2>
-        <DowntimeNotification
-          appTitle="Military Information"
-          render={handleDowntimeForSection('military service')}
-          dependencies={[externalServices.emis]}
+  return (
+    <>
+      <ProfileInfoTable
+        data={serviceHistory}
+        dataTransformer={transformServiceHistoryEntryIntoTableRow}
+        title="Period of service"
+        fieldName="serviceHistory"
+      />
+      <div className="vads-u-margin-top--4 vads-u-margin-bottom--6">
+        <AdditionalInfo
+          triggerText="What if my military service information doesn’t look right?"
+          onClick={() => {
+            recordEvent({
+              event: 'profile-navigation',
+              'profile-action': 'view-link',
+              'profile-section': 'update-military-information',
+            });
+          }}
         >
-          <MilitaryInformationContent
-            militaryInformation={this.props.militaryInformation}
-          />
-        </DowntimeNotification>
-      </>
-    );
-  }
-}
+          <p>
+            Some Veterans have reported seeing military service information in
+            their VA.gov profiles that doesn’t seem right. When this happens,
+            it’s because there’s an error in the information we’re pulling into
+            VA.gov from the Defense Enrollment Eligibility Reporting System
+            (DEERS).
+          </p>
+          <p>
+            If the military service information in your profile doesn’t look
+            right, please call the Defense Manpower Data Center (DMDC). They’ll
+            work with you to update your information in DEERS.
+          </p>
+          <p>
+            To reach the DMDC, call{' '}
+            <a
+              href="tel:1-800-538-9552"
+              aria-label="8 0 0. 5 3 8. 9 5 5 2."
+              title="Dial the telephone number 800-538-9552"
+              className="no-wrap"
+            >
+              1-800-538-9552
+            </a>
+            , Monday through Friday (except federal holidays), 8:00 a.m. to 8:00
+            p.m. ET. If you have hearing loss, call TTY: 1-866-363-2883.
```

Comment:
Should this phone number be wrapped in an `<a>` tag as well?

Reply:
I'm not sure if we do that with TTY numbers. I'll check.

Reply:
we are definitely not consistent with how we handle this, but sometimes we do wrap it.